test/util/Tokenizer.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { expect } from "chai";
+import {Tokenizer, TokenizerError} from "../../src/util/Tokenizer";
+
+describe("Tokenizer tokenize", () => {
+    it("gets and checks tokens in projectstructureex", () => {
+        let tokenizer: Tokenizer = new Tokenizer("projectstructureex.txt", "./test/testFiles");
+        //dir src
+        tokenizer.getNext();
+        expect(tokenizer.getNext()).to.equal("dir");
+        expect(tokenizer.getNext()).to.equal("src");
+        // 	class Time
+        tokenizer.getNext();
```

Comment:
this should be an INDENT_LEVEL = 8 right? 

dont have to change it, just checking my understanding of how you're dealing with tabs

Reply:
yep- should be `_INDENT_LEVEL=8_`
One `\t` will increase the space count by 8